docs/manual/mkdocs.yml

```diff
@@ -91,6 +91,7 @@ nav:
         - navigation node remove: 'cmd/spo/navigation/navigation-node-remove.md'
       - page:
         - page add: 'cmd/spo/page/page-add.md'
+        - page remove: 'cmd/spo/page/page-remove-md'
```

Comment:
Entries are sorted alphabetically, so ideally `page remove` should go between `page list` and `page set`.

Reply:
Done